build.rs

```diff
@@ -26,13 +26,13 @@ fn main() {
 
         gcc::Build::new()
             .cpp(true)
-            .debug(true)
+            //.debug(true)
```

Comment:
I prefer just delete unneeded code instead of commenting it out. It lives forever in Git anyway 🙂